src/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/Debug/LaunchSettingsProviderTests.cs

```diff
@@ -458,7 +458,7 @@ public async Task LaunchSettingsFile_TestIgnoreFlag()
                 // Set the ignore flag. It should be ignored.
                 provider.LastSettingsFileSyncTimeTest = DateTime.MinValue;
                 provider.SetIgnoreFileChanges(true);
-                Assert.Null(provider.LaunchSettingsFile_ChangedTest());
+                Assert.Equal(provider.LaunchSettingsFile_ChangedTest(), Task.CompletedTask);
```

Comment:
I don't think equality checks really make sense for tasks. I'd either make this `Assert.Same`, or test the returned task's state via its properties.